blueprints.yaml

```diff
@@ -321,4 +321,23 @@ form:
       label: Visitors history
       default: 20
 
+    Editor:
+      type: section
+      title: Editor
```

Comment:
Yeah, in my mind this "Editor" section could eventually have properties for all parts of the editor, including the Content part of it.

It seems a bit weird to me to have a config section specifically for a sub-part of a component, rather than for the whole component.

Happy to update if you're sure, though